src/components/Textarea/index.tsx

```diff
@@ -96,6 +96,12 @@ export interface TextareaProps {
     onFocus?: (event: React.FocusEvent<HTMLTextAreaElement>) => void;
     /** Handler for the onBlur event */
     onBlur?: (event: React.FocusEvent<HTMLTextAreaElement>) => void;
+    /** Handler for the onKeyDown event */
+    onKeyDown?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void;
+    /** Handler for the onKeyUp event */
+    onKeyUp?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void;
+    /** Additional classes to apply */
+    classes?: string[];
```

Comment:
At NorthStar, we usually try to avoid custom override of styling to ensure the gaps in the design system are surfaced. Usually, we use `variant` to offer different variants of the components. 

If you feel that your styling can be useful for other people, you can create a `variant`. Otherwise, use the  
`className | string | Adds the specified classes to the root element of the component`
as API.

Reply:
Will change to className